### PR TITLE
authorization code grant flow

### PR DESCRIPTION
**Pull request by nicolasparada**
Changing the device code flow for authorization code grant flow.
Closes https://github.com/calyptia/core-docker-desktop-extension/issues/73

https://user-images.githubusercontent.com/7969166/216369757-71fdca4d-83c5-481f-961a-86700381ed5f.mov

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,12 @@ install-extension: build-extension ## Install the extension
 update-extension: build-extension ## Update the extension
 	docker extension update $(IMAGE):$(TAG) --force
 
+dev-ui: ## Make sure to run `cd ui && npm start` firts.
```
Comment (patrick-stephens) [inline Makefile:18]: Typo of first

File: ui/.env [hunk 2]
```diff
@@ -1,6 +1,6 @@
 PUBLIC_URL=.
 BROWSER=none
 REACT_APP_AUTH0_DOMAIN=sso.calyptia.com
-REACT_APP_AUTH0_CLIENT_ID=vsywuHQDbLFrz9TFHyLMXskZcSclHxEd
+REACT_APP_AUTH0_CLIENT_ID=hODtvC3oOQVwT8ILxoQFv5eRtgElsdFh
```
Comment (patrick-stephens) [inline ui/.env:4]: I presume this is ok to commit, should it be a secret?

Comment (nicolasparada) [reply ui/.env:4]: It's not a secret. I can put it as a default instead and remove the .env file from git.

Comment (nicolasparada) [reply ui/.env]: Done.

File: Makefile (hunk 1 above)
Comment (nicolasparada) [reply Makefile]: Fixed

**Review by patrick-stephens — APPROVED** [review]

**Comment by edsiper** [comment]
thanks

**Pull request merged**